packages/TagPicker/src/TagPicker.tsx

```diff
@@ -184,7 +184,7 @@ const TagPicker: React.FunctionComponent<TagPickerProps> = ({
         onSelection(resultId);
         resetSearch();
 
-        if (maxTags && selectedResultsCount === maxTags - 1) {
+        if (maxTags && selectedResultsCount >= maxTags - 1) {
```

Comment:
thats an odd change to the check. 

before: "If there is only 1 tag spot left, then do this"
now: "If there are more tags than allowed, then do this". Why is this possible? 

It feels like treating the symptoms more than the cause of the issue

Reply:
Indeed, upon further exploration the -1 does not seems to make sense at all. As for the > this was pretty much a defensive way of making sure that if we somehow have more than the max it will be flagged.